InkScripts/FabledLandsparagraphs_patapon4321.ink

```diff
@@ -0,0 +1,1897 @@
+VAR trubadour = false
+VAR wayfarer = false
+VAR warrior = false
+VAR mage = false
+VAR rogue = false
+VAR priest = false
+VAR inititate = false
+VAR deliver = false
+VAR Shards = 0
+VAR trident = false
+VAR climbing_gear = false
+VAR scouting = 0
+VAR altitude = false
+VAR assassin = false
+VAR anchor = false
+VAR avenge = false
+VAR stamina = 0
+VAR rank = 0
+VAR visited_paragraph_16 = false
```

Comment:
Inky have built-in functionality to track if knob/stitch was visited and how many times.
Please read more details here https://github.com/inkle/ink/blob/master/Documentation/WritingWithInk.md#conditional-choices